build.gradle

```diff
@@ -1,4 +1,5 @@
 plugins {
+    id 'waifu-motivator-build-plugin'
```

Comment:
This comes from the [buildSrc](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources)